File: web3rpc/rpc-specs/code-samples/python/klay/block/getBlockByNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x1b4"
boolean = True

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_block_by_number(blockTag, boolean)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getBlockByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumber = 1
    const returnTransactionObject = true
    
    provider.klay.getBlockByNumber(blockNumber, returnTransactionObject, {}, (err, data, response) => {
        console.log(data);
    });
}
)()